code/__HELPERS/_lists.dm

```diff
@@ -50,7 +50,7 @@
 			var/__BIN_LEFT = 1;\
 			var/__BIN_RIGHT = __BIN_CTTL;\
 			var/__BIN_MID = (__BIN_LEFT + __BIN_RIGHT) >> 1;\
-			var/##TYPECONT/__BIN_ITEM;\
+			var ##TYPECONT/__BIN_ITEM;\
```

Comment:
Was this intended? Doesn't look right. Maybe I'm missing something though.

Reply:
Yes appears to be a by-product of https://github.com/tgstation/tgstation/pull/53217

Reply:
Kind of weird but alright, seems fine then.